Modules/Loadable/Segmentations/Logic/vtkSlicerSegmentationGeometryLogic.cxx

```diff
@@ -127,7 +127,6 @@ std::string vtkSlicerSegmentationGeometryLogic::CalculateOutputGeometry()
   // Determine source type
   vtkMRMLScalarVolumeNode* sourceVolumeNode = vtkMRMLScalarVolumeNode::SafeDownCast(this->SourceGeometryNode);
   vtkMRMLAnnotationROINode* sourceRoiNode = vtkMRMLAnnotationROINode::SafeDownCast(this->SourceGeometryNode);
-  vtkMRMLSegmentationNode* sourceSegmentationNode = vtkMRMLSegmentationNode::SafeDownCast(this->SourceGeometryNode);
 
   if (sourceVolumeNode || this->IsSourceSegmentationWithBinaryLabelmapMaster())
```

Comment:
```suggestion
  if (sourceVolumeNode
    || (sourceSegmentationNode && this->IsSourceSegmentationWithBinaryLabelmapMaster()))
```

Reply:
Thanks for the review. I updated the commit message and implemented your suggestion.

Also, since the function `IsSourceSegmentationWithBinaryLabelmapMaster` already include the check ... could you confirm that the suggested change is to improve readability of the current code and minimize introduction of issue when working on the code later on ?

https://github.com/Slicer/Slicer/blob/6751d8e6d8cdc8e7c221ed2165d902cd0c066554/Modules/Loadable/Segmentations/Logic/vtkSlicerSegmentationGeometryLogic.cxx#L349-L364